scripts/pre-deployment-test.ts

```
@@ -0,0 +1,264 @@
+      'knowledge_base_v2',
+      'rag_implementation_metrics',
+      'performance_baselines',
+    ];
+    
+    for (const table of tables) {
+      const { error } = await supabaseAdmin
+        .from(table)
+        .select('*')
+        .limit(1);
+      
+      if (error && error.message.includes('does not exist')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)